brambling/models.py

```diff
@@ -2,7 +2,7 @@
 from __future__ import unicode_literals
 
 from collections import OrderedDict
-from datetime import timedelta
+from datetime import date as dtdate, timedelta
```

Comment:
Non-blocking, but elsewhere in the repo and in other projects I don't think we've bothered to qualify the `datetime.date` import